content/de/docs/concepts/cluster-administration/addons.md

```diff
@@ -38,7 +38,7 @@ Die Add-Ons in den einzelnen Kategorien sind alphabetisch sortiert - Die Reihenf
 
 * [CoreDNS](https://coredns.io) ist ein flexibler, erweiterbarer DNS-Server der in einem Cluster [installiert](https://github.com/coredns/deployment/tree/master/kubernetes) werden kann und das Cluster-interne DNS für Pods bereitzustellen.
 
-## Visualization &amp; Control
+## Visualisierung &amp; Überwachung
```

Comment:
@sftim I think this is better than keeping the heading in english.

Reply:
Fine by me!